dnf-behave-tests/features/plugins-core/needs-restarting-conf-files.feature

```diff
@@ -0,0 +1,122 @@
+Feature: Add package to needs-restarting using config files
+
+Background:
+    Given I enable plugin "needs_restarting"
+      And I use repository "dnf-ci-fedora"
+      And I move the clock backward to "before boot-up"
+      And I execute dnf with args "install lame kernel basesystem glibc wget abcde"
```

Comment:
Looks like this list of packages is unnecessary, you're only ever using wget and abcde from the list?

While not necessary, I'd prefer if you set up a single repository with the 3 packages you're using and used only that in the feature file, to make it a bit cleaner... I'm not requiring it though.